### ISSUE TITLE
No -init-driver function found for 'metabase.driver.sparksql'

### ISSUE THREAD

**Issue by witwall** [metabase/metabase#7536]
------
Before filing an issue we'd appreciate it if you could take a moment to ensure
there isn't already an open issue or pull-request.
-----

If there's an existing issue, please add a :+1: reaction to the description of
the issue. One way we prioritize issues is by the number of :+1: reactions on
their descriptions. Please DO NOT add `+1` or :+1: comments.

### Database support

For requests for new database drivers, please include the name
of the database and the version. Additionally, giving us a sense of how and what
you use it for, and other high-level details about the database, can help us get
a better picture of what would need to done to support it.

### Feature requests and proposals
We're excited to hear how we can make Metabase better. Please add as much detail
as you can on your use case.

### Bugs
If you're filing an issue about a bug please include as much information
as you can including the following.

- Your browser and the version: (e.x. Chrome 52.1, Firefox 48.0, IE 10)
Chrome 66.0.3359.117
- Your operating system: (e.x. OS X 10, Windows XP, etc)
OS X 10
- Your databases: (e.x. MySQL, Postgres, MongoDB, …)
H2
- Metabase version: (e.x. 0.19.3)
v0.29.0-RC1 (95a6267 release-0.29.0)
- Metabase hosting environment: (e.x. Mac app, Elastic Beanstalk, Docker, Heroku, Linux/Ubuntu 12)
Mac app
- Metabase internal database: (e.x. H2 (default), MySQL, Postgres)
H2
- *Repeatable steps to reproduce the issue*

run spark first,

then run

```
# Other Java options
JAVA_OPTS="$JAVA_OPTS -XX:+IgnoreUnrecognizedVMOptions"           # Don't barf if we see an option we don't understand (e.g. Java 9 option on Java 7/8)
JAVA_OPTS="$JAVA_OPTS -Djava.awt.headless=true"                   # don't try to start AWT. Not sure this does anything but better safe than wasting memory
JAVA_OPTS="$JAVA_OPTS -Dfile.encoding=UTF-8"                      # Use UTF-8
JAVA_OPTS="$JAVA_OPTS --add-opens=java.base/java.net=ALL-UNNAMED" # Allow dynamically adding JARs to classpath (Java 9)
JAVA_OPTS="$JAVA_OPTS --add-modules=java.xml.bind"                # Enable access to java.xml.bind module (Java 9)

echo "Using these JAVA_OPTS: ${JAVA_OPTS}"

export MB_JETTY_PORT=8000
export MB_JETTY_HOST=0.0.0.0

java $JAVA_OPTS -cp/Users/steven/spark/spark-2.2.0-bin-hadoop2.7/jars/* -jar metabase.jar
```
Thanks for being part of the Metabase project!
-------

**Comment by camsaul** [comment]
Oops. I fixed this in #7503 but forgot to merge. I'll go ahead and merge that now

**Comment by camsaul** [comment]
Fixed by #7503
